src/organization/organization.route.ts

```diff
@@ -48,6 +48,19 @@ function organizationRouterConfig($stateProvider) {
         }
       }
     })
+    .state('organization.settings.cockpit', {
+      url: '/cockpit',
+      component: 'cockpit',
+      data: {
+        menu: null,
+        docs: {
+          page: 'organization-configuration-cockpit'
```

Comment:
You use "settings" for the state, configuration for the contextual doc and installation for the perm. Can we use a single term?

Reply:
I did exactly the same as the others pages. For instance, to access categories page:
- the state is `management.settings.categories`
- the doc is `management-configuration-categories`
- the perm is `environment-category-r`